candidate_num: 50
corner_thre: 10
descriptor_max_len: 50
descriptor_min_len: 2
descriptor_near_num: 10
dis_threshold: 0.5
ds_size: 0.25
icp_threshold: 0.4
is_benchmark: false
lidar_path: ''
maximum_corner_num: 100
non_max_suppression_radius: 2
normal_threshold: 0.2
plane_detection_thre: 0.01
plane_merge_normal_thre: 0.2
pose_path: ''
proj_dis_max: 5
proj_dis_min: 0
proj_image_resolution: 0.5
rough_dis_threshold: 0.01
seq_name: ''
skip_near_num: 50
std_side_resolution: 0.2
sub_frame_num: 10
vertex_diff_threshold: 0.5
voxel_init_num: 10
voxel_size: 2.0
